src/yardstick/artifact.py

```diff
@@ -499,6 +500,15 @@ def effective_year(self, by_cve=False) -> Optional[int]:
         return year
 
 
+class LabelEntryCollection:
```

Comment:
I'm confused. Is this meant to be used as the type of `label_entries` in `src/yardstick/__init__.py`?

Reply:
not in this PR, no. This PR only adds this object to be used by API consumers. In the future this can be rolled underneath the core API.